releasenotes/notes/tomography-b091ce13d6983bc1.yaml

```
@@ -5,6 +5,38 @@ features:
     :class:`~.StateTomography` and :class:`~.ProcessTomography` experiments.
     These allow specifying intended backend, a custom analysis class, or a
     custom target for fidelity calculations, when initializing the experiments.
+  - |
+    Improves :class:`.LocalMeasurementBasis` and `.LocalPreparationBasis`
+    tomography basis classes support for initializing a noisy basis for
+    performing state preparation and measurement error mitigated
+    :class:`.StateTomography` and :class:`.ProcessTomography` experiments.
+
+    For preparation bases reset noise in a specific qubit, or subset of qubits,
+    can now be input as a quantum channel, and the noisy prepared states are
+    generated by applying the ideal instructions to the noisy initial state.
```

Difficult to parse. Suggestion:
```suggestion
    For preparation bases, a noisy reset operation on a specific qubit,
    or subset of qubits, can now be input as a quantum channel, and the
    noisy prepared states are generated by applying the ideal instructions
    to the noisy initial state.
```